spacemacs.org

```diff
@@ -0,0 +1,97 @@
+* Why I use Spacemacs
+
+Software engineering is a craft, and one of the most important thing to any
+craftman (or craftwoman!), is their tools. Editor choice is a very personal and
```

Comment:
craftman/craftwoman -> craftsman/craftswoman. 
Probably get rid of comma.
is their tools -> is their choice of tools (to match plural/singular better, slight repetition is okay)